code/game/objects/items/weapons/tools.dm

```diff
@@ -358,7 +358,7 @@
 	var/safety = user:eyecheck()
 	if(ishuman(user))
 		var/mob/living/carbon/human/H = user
-		var/obj/item/organ/eyes/E = H.internal_organs_by_name["eyes"]
+		var/obj/item/organ/internal/eyes/E = H.get_int_organ(/obj/item/organ/internal/eyes)

```

Comment:
the vision organ and path for ipcs is /obj/item/organ/internal/optical_sensor @tigercat2000 @Crazylemon64 @Fox-McCloud howfix/shouldfix?